### ISSUE TITLE
FORMULA: Request related to a bottle: CRC

### ISSUE THREAD

**Issue by gbraad** [Homebrew/homebrew-core#54556]
This relates to: https://formulae.brew.sh/formula/crc

When checking the following code:
https://github.com/Homebrew/homebrew-core/blob/master/Formula/crc.rb
I fail to see how maintains it or proposed this.

By having this being installable from brew we receive requests related to the missing parts: https://github.com/code-ready/crc/issues/1228 like the image bundle. These can not be redistributed without people accepting the terms for the RedHat Developer program. In other words, the binary by itself is not usable.

When an upstream version is available we will pick this up and allow for a brew installable version... but at this point this does not make a lot of sense.

**Comment by SMillerDev** [comment]
Made a PR to remove CRC. Everyone can edit any homebrew formula but the initial commit was here: https://github.com/Homebrew/homebrew-core/commit/4edc8851938336099895e0caa321c2c97ea4b7cb

**Comment by iMichka** [comment]
If I understand well, crc is open source (Apache License 2.0), but there is an image bundle which is not. We do not ship the bundle, making crc useless?

**Comment by gbraad** [comment]
TL;dr Yes, kinda.

The current CRC is the downstream product to offer an OpenShift
cluster based on OCP, and this bundle can not be shipped without
accepting the terms. The upstream implementation, which will be based
on the same Apache -licensed codebase, will come to have a freely
redistributable imagebundle that will likely be hosted on
mirror.openshift.com. So, yes... the current command is useless, and
unsupported by us.

On Mon, May 11, 2020 at 6:20 PM Michka Popoff <notifications@github.com> wrote:
>
> If I understand well, crc is open source (Apache License 2.0), but there is an image bundle which is not. We do not ship the bundle, making crc useless?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.



-- 

   Gerard Braad | http://gbraad.nl
   [ Doing Open Source Matters ]

**Comment by gbraad** [comment]
@cblecker what was the purpose of the brew version when no imagebundle is generally available? It would mean to make use of the current command you either used the version from https://mirror.openshift.com/pub/openshift-v4/clients/crc/latest/ or you got access to the internal image builds. For general use, this would cause confusion and more work for us (/me is teamlead of CRC)
